src/main/java/kafdrop/util/Deserializers.java

```diff
@@ -0,0 +1,28 @@
+package kafdrop.util;
+
+public class Deserializers {
+
+    private MessageDeserializer keyDeserializer;
```

Comment:
Do we ever need to set these deserializers other than in the constructor? If not, can we make them `final`?

Reply:
Agree. Resolved in latest push